### ISSUE TITLE
How to use

### ISSUE THREAD

**Issue by kody435** [coreyward/sanity-image#19]
Sir, I saw you on the YouTube video, where you explained how it will work. I want to know how can I use it in my sanity studio.

Thank You
Param

**Comment by coreyward** [comment]
Hey Kody, thanks for the reminder! I found a few developer-ergonomics issues and some edge-case bugs that I am ironing out as I write tests. Once those are done and I'm confident with the API I'll get documentation drafted up and shared. 

If you really want to play with it as-is knowing the props API will change, here's an example of how to use the alpha version:

```tsx
import { SanityImage, createBuilder } from "sanity-image"

// configure the image-url builder
const builder = createBuilder({
  dataset: "your-dataset",
  projectId: "your-project-id",
})

// build a wrapper component so you don't have to pass the builder every time you use it
const Image = (props) => (
  <SanityImage builder={builder} {...props} />
);

// use it
const SomeComponent = ({ sanityImageId }) => (
  <Image
    // optionally, do you want to use a custom component instead of the default img element?
    as="img"

    // asset object with `_id` (or `_ref`) is required; metadata is optional
    asset={{
      _id: sanityImageId,
      metadata: {
        preview: "base64 encoded preview image, if desired/available",
      },
    }}
    hotspot={{
      // hotspot data if available
    }}
    crop={{
      // crop data if available
    }}

    // how wide are you expecting to show the image? this will inform the srcset, but does not set any styles for you
    width={500}
    height={500} // optional, generally not recommended unless you want a specific aspect ratio

    // automatically add `width` and `height` attributes to the `img` tag, if desired
    options={{ aspectRatio: true }}

    // any other img element props you want, like `className` or `style` or `sizes`
  />
)
```

**Comment by coreyward** [comment]
Just shipped `v0.1.1` including a README file. Please check it out and let me know if anything isn't clear!
